src/main/java/com/minecolonies/coremod/colony/jobs/JobDeliveryman.java

```diff
@@ -206,9 +229,9 @@ public void onTaskDeletion(@NotNull final IToken token)
      *
      * @return The task queue.
      */
-    public List<IToken> getTaskQueue()
+    public List<IToken<?>> getTaskQueue()
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove usage of generic wildcard type. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1452)